operator/main.go

```diff
@@ -154,7 +155,7 @@ func setupWithManager(flagVar *FlagVar, newCacheFunc cache.NewCacheFunc, scheme
 		setupLog.Error(err, "unable to create controller", "controller", "Manifest")
 		os.Exit(1)
 	}
-	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+	if os.Getenv("ENABLE_WEBHOOKS") == "true" {
```

Comment:
This disables activating webhooks with 
```sh
export ENABLE_WEBHOOKS
```
do we want this?

Reply:
it was an interim solution, I will switch to a flag